src/bin/daemon.rs

```diff
@@ -5,27 +5,37 @@ use std::{
     process, thread, time,
 };
 
-use liana::{config::Config, DaemonHandle};
+use liana::{config::Config, DaemonHandle, VERSION};
 
-fn print_help_exit() {
+fn print_help_exit(code: i32) {
+    eprintln!("lianad version {}", VERSION);
     eprintln!("A TOML configuration file is required to run lianad. By default lianad looks for a 'config.toml' file in its data directory. A different one may be provided like so: '--conf <config file path>'.");
     eprintln!("A documented sample is available at 'contrib/lianad_config_example.toml' in the source tree (https://github.com/wizardsardine/liana/blob/v1.0/contrib/lianad_config_example.toml).");
     eprintln!("The default data directory path is a 'liana/' folder in the XDG standard configuration directory for all OSes but Linux ones, where it's '~/.liana/'.");
-    process::exit(1);
+    process::exit(code);
+}
+
+fn print_version() {
+    eprintln!("{}", VERSION);
+    process::exit(0);
 }
 
 fn parse_args(args: Vec<String>) -> Option<PathBuf> {
     if args.len() == 1 {
         return None;
     }
 
-    if args[1] != "--conf" {
+    if args[1] == "--help" || args[1] == "-h" {
```

Comment:
If anybody looks at this and thinks it does not do proper bounds checks, the args.len() check above actually does. This is because args is never empty because it always contains the program's name. Still i keep having to double check this everytime i read this code so it'd be worth making the check above `args.len() <= 1`.